test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1

```diff
@@ -339,4 +339,141 @@ Left Center Right
 			$output = $ps.Invoke()
 			$output.Replace("`n","").Replace("`r","") | Should -BeExactly $expected
 		}
+
+		It "Format-Table should correctly render headers that span multiple rows: <variation>" -TestCases @(
+			@{ view = "Default"; widths = 4,7,4; variation = "4 row, 1 row, 2 row"; expectedTable = @"
+
+Long Header2 Head
+Long         er3
+Head
+er
+---- ------- ----
+1    2       3
+
+
+
+"@ },
+			@{ view = "Default"; widths = 4,4,7; variation = "4 row, 2 row, 1 row"; expectedTable = @"
+
+Long Head Header3
+Long er2
+Head
+er
+---- ---- -------
+1    2    3
+
+
+
+"@ },
+			@{ view = "Default"; widths = 14,7,3; variation = "1 row, 1 row, 3 row"; expectedTable = @"
+
+LongLongHeader Header2 Hea
+                       der
+                       3
+-------------- ------- ---
+1              2       3
+
+
+
+"@ },
+			@{ view = "One"; widths = 4,1,1; variation = "1 column"; expectedTable = @"
+
+Long
+Long
+Head
+er
+----
+1
+
+
+
+"@ }
+		) {
+			param($view, $widths, $expectedTable)
+			$ps1xml = @"
+<Configuration>
+	<ViewDefinitions>
+		<View>
+			<Name>Default</Name>
+			<ViewSelectedBy>
+				<TypeName>Test.Format</TypeName>
+			</ViewSelectedBy>
+			<TableControl>
+				<TableHeaders>
+					<TableColumnHeader>
+						<Label>LongLongHeader</Label>
+						<Width>{0}</Width>
+					</TableColumnHeader>
+					<TableColumnHeader>
+						<Label>Header2</Label>
+						<Width>{1}</Width>
+					</TableColumnHeader>
+					<TableColumnHeader>
+						<Label>Header3</Label>
+						<Width>{2}</Width>
+					</TableColumnHeader>
+				</TableHeaders>
+				<TableRowEntries>
+					<TableRowEntry>
+						<TableColumnItems>
+							<TableColumnItem>
+								<PropertyName>First</PropertyName>
+							</TableColumnItem>
+							<TableColumnItem>
+								<PropertyName>Second</PropertyName>
+							</TableColumnItem>
+							<TableColumnItem>
+								<PropertyName>Third</PropertyName>
+							</TableColumnItem>
+						</TableColumnItems>
+					</TableRowEntry>
+				</TableRowEntries>
+			</TableControl>
+		</View>
+		<View>
+			<Name>One</Name>
+			<ViewSelectedBy>
+				<TypeName>Test.Format</TypeName>
+			</ViewSelectedBy>
+			<TableControl>
+				<TableHeaders>
+					<TableColumnHeader>
+						<Label>LongLongHeader</Label>
+						<Width>{0}</Width>
+					</TableColumnHeader>
+				</TableHeaders>
+				<TableRowEntries>
+					<TableRowEntry>
+						<TableColumnItems>
+							<TableColumnItem>
+								<PropertyName>First</PropertyName>
+							</TableColumnItem>
+						</TableColumnItems>
+					</TableRowEntry>
+				</TableRowEntries>
+			</TableControl>
+		</View>
+	</ViewDefinitions>
+</Configuration>
+"@
+			$ps1xml = $ps1xml.Replace("{0}", $widths[0]).Replace("{1}", $widths[1]).Replace("{2}", $widths[2])
+			$ps1xmlPath = Join-Path -Path $TestDrive -ChildPath "test.format.ps1xml"
+			Set-Content -Path $ps1xmlPath -Value $ps1xml
```

Comment:
Maybe add `-Force`?

Reply:
Isn't needed here, `-Force` is only needed if the file is read-only